breol_news_page/breol_news_page.module

```diff
@@ -51,3 +51,27 @@ function breol_news_page_theme() {
     ),
   );
 }
+
+/**
+ * Implements hook_preprocess_entity().
+ */
```

Comment:
A extra comment about why this is needed would be nice?

Reply:
@cableman , I couldn't agree more. However, this is lifted from [`reol_news_paragraph_preprocess_entity`](https://github.com/eReolen/base/blob/develop/sites/all/modules/paragraphs/reol_news_paragraph/reol_news_paragraph.module#L97-L115) which could also use a clarifying comment …

Reply:
I've added a comment that hopefully contributes with some useful information.